support-models/src/main/scala/com/gu/support/workers/states/SendThankYouEmailState.scala

```diff
@@ -11,6 +11,7 @@ case class SendThankYouEmailState(
   paymentMethod: PaymentMethod,
   salesForceContact: SalesforceContactRecord,
   accountNumber: String,
```

Comment:
Account number is still currently required for fetching direct debit details: https://github.com/guardian/support-workers/blob/master/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala#L37-L41